src/frontend/js/components/OpenEdxFullNameForm/index.tsx

```diff
@@ -79,10 +76,12 @@ const OpenEdxFullNameForm = () => {
     reValidateMode: 'onChange',
     resolver: yupResolver(validationSchema),
   });
+
   const { register, handleSubmit, reset, formState } = form;
 
-  const onSubmit = (values: OpenEdxFullNameFormValues) => {
-    update(values);
+  const onSubmit = async (values: OpenEdxFullNameFormValues) => {
+    await update(values);
+    buttonRef.current?.blur();
```

Comment:
IMO, when the username is updated, the default Values should be updated so the submit should be disabled. Currently this is weird as there is no feedback that the user full name has been updated.